lib/renderer/window-setup.ts

```diff
@@ -233,7 +233,8 @@ export const windowSetup = (
   defineProperty(window.history, 'length', {
     get: function () {
       return ipcRendererInternal.sendSync('ELECTRON_NAVIGATION_CONTROLLER_LENGTH')
-    }
+    },
+    configurable: true
```

Comment:
The behavior in browsers isn't that this is configurable, if you do `window.history.length = 100` in Chrome it doesn't throw but `window.history.length` doesn't change.

We should instead make a noop-setter here.
```suggestion
    set() {}
```